### PR TITLE
Added slider for Brightness and Blur modules

### LINKED ISSUE #423: add slider UI to some modules HTML UI: brightness, blur
I believe there's a few simple slider libraries available! jQuery slider, and you can probably search https://npmjs.com for more options. 

Let's start with the brightness module and make adjusting brightness easier for people!

### PR DESCRIPTION
**Pull request by aashna27**
Fix #423 
Sliders were added to Brightness and Blur modules.
image

image

### CODE REVIEW

**Comment by jywarren** [comment]
Oh, this is awesome! Is it possible to preserve the numerical input as
well, and have them stay synced?

Thank you!!!!!

**Comment by aashna27** [comment]
Does it mean, providing the user with 2 input options for the same field in these modules.
Wouldn't this lead to unnecessary redundancy? The slider is easy to use even for really close values.

**Comment by jywarren** [comment]
Hmm, i guess it's OK this way. I was thinking that the slider may be showing only recommended ranges, and people could conceivably want to input their own. But I think you're right that it's a pretty low probability. 

Have you confirmed that pressing Save and then changing it again and pressing Save a second time works well? Thank you!!!

**Comment by aashna27** [comment]
Okay! Lets make it with the slider only. :smiley: . And yes i did verify that the slider works even after clicking save. 😄

File: examples/lib/defaultHtmlStepUi.js [hunk 1]
```diff
@@ -75,8 +75,20 @@ function DefaultHtmlStepUi(_sequencer, options) {
             '" value="' +
             paramVal +
             '" placeholder ="' +
-            (inputDesc.placeholder || "") +
-            '">';
+            (inputDesc.placeholder || "");
+            
+           if(inputDesc.type.toLowerCase() == "range")
```
Comment (jywarren) [inline examples/lib/defaultHtmlStepUi.js]: Oh wow, I love this implementation! What a great way to do it!!!! 🎉 @tech4GT check it out :-)

File: src/modules/Blur/info.json [hunk 2]
```diff
@@ -3,9 +3,12 @@
...
     "inputs": {
         "blur": {
-            "type": "integer",
+            "type": "range",
             "desc": "Amount of gaussian blur(Less blur gives more detail, typically 0-5)",
-            "default": 2
+            "default": "2",
+            "min": "0",
+            "max": "5",
+            "step": "1"
```
Comment (jywarren) [inline src/modules/Blur/info.json]: Here can we make it step by 0.25 maybe?

File: examples/lib/defaultHtmlStepUi.js (hunk 1 above)
Comment (aashna27) [reply examples/lib/defaultHtmlStepUi.js]: Thanks 😸

File: src/modules/Blur/info.json (hunk 2 above)
Comment (aashna27) [reply src/modules/Blur/info.json]: Yes, sure !! I ll do it.

**Comment by jywarren** [comment]
This is perfect. Now, we need to compile the changes into the `/dist/ ` directory -- you can do this by running `grunt build` in the root directory. Then check in the files generated into `/dist/...` and add them to this PR. 

If you get stuck I can help! Thank you!

**Comment by jywarren** [comment]
Great work!!!

**Comment by jywarren** [comment]
Hi @aashna27 -- would you be interested in doing a follow-up PR where brightness could be turned up to 200, instead of 100 being the limit?

Also, I noticed that when first run, it seems to be at 0%, but it shows up all white -- maybe we should have the default be 100%? 

Thank you in any case, this is really cool!

http://sequencer.publiclab.org/examples/#steps=brightness

image

**Comment by aashna27** [comment]
@jywarren I would look into it. Thanks

**Comment by jywarren** [comment]
oh awesome, thank you!

**Pull request merged**